VirtoCommerce.Domain/Catalog/Model/Search/SearchCriteria.cs

```diff
@@ -1,11 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
+using System;
 using System.Linq;
+using VirtoCommerce.Domain.Commerce.Model.Search;
 using VirtoCommerce.Platform.Core.Common;
 
 namespace VirtoCommerce.Domain.Catalog.Model
 {
-    public class SearchCriteria
+    public class SearchCriteria : SearchCriteriaBase
```

Comment:
It is the breaking changes. I guess we can limit ourselves only inheritance from ValueObject there.